src/js/tech/html5.js

```diff
@@ -202,7 +204,30 @@ class Html5 extends Tech {
       // source elements will be used but implementing the source selection algorithm
       // is laborious and asynchronous, so,
       // instead return an empty string to basically indicate source may change
-      this.triggerSourceset(el.src || '');
+      let src = el.src || '';
+
+      // if their is only one source
```

Comment:
Realized that we can actually know what `<source>` it will choose if there isn't a `src` attribute, and there is only one non-duplicate `<source>` url.